src/Routes.js

```diff
@@ -37,117 +33,66 @@ const TopicAdmin = lazy(() =>
     /* webpackChunkName: "TopicAdmin" */ './PresentationalComponents/TopicsAdminTable/TopicsAdminTable'
   )
 );
-const AdminProtectedRoute = lazy(() =>
-  import(
-    /* webpackChunkName: "TopicAdmin" */ './PresentationalComponents/TopicsAdminTable/TopicsAdminTable'
-  )
-);
-
-const paths = [
-  //Recommendations nav -> recommendations tab is active
-  { title: 'Recommendations', path: '/recommendations', component: RecsList },
-  //Recommendations nav -> pathways tab is active
-  {
-    title: 'Recommendations Pathways',
-    path: '/recommendations/pathways',
-    component: RecsList,
-  },
-  //Pathway details =>  recommendations tab is active
-  {
-    title: 'Pathway details',
-    path: '/recommendations/pathways/:id',
-    component: DetailsPathways,
-  },
-  //Pathway details =>  systems tab is active
-  {
-    title: 'Pathway details',
-    path: '/recommendations/pathways/systems/:id',
-    component: DetailsPathways,
-  },
-  {
-    title: 'Recommendation details',
-    path: '/recommendations/:id',
-    component: RecsDetails,
-  },
-  {
-    title: 'Inventory details',
-    path: '/recommendations/:id/:inventoryId/',
-    component: InventoryDetails,
-  },
-  {
-    title: 'Inventory details',
-    path: '/recommendations/pathways/:id/:inventoryId/',
-    component: InventoryDetails,
-  },
-  { title: 'Systems', path: '/systems', component: SystemsList },
-  {
-    title: 'System detail',
-    path: '/systems/:inventoryId/',
-    component: InventoryDetails,
-  },
-  { title: 'Topics', path: '/topics', component: TopicsList },
-  { title: 'Topic details', path: '/topics/:id', component: TopicDetails },
-];
-const INVENTORY_TOTAL_FETCH_URL = '/api/inventory/v1/hosts';
 
-export const Routes = () => {
-  const [hasSystems, setHasSystems] = useState(true);
-  useEffect(() => {
-    try {
-      axios
-        .get(`${INVENTORY_TOTAL_FETCH_URL}?page=1&per_page=1`)
-        .then(({ data }) => {
-          setHasSystems(data.total > 0);
-        });
-    } catch (e) {
-      console.log(e);
-    }
-  }, [hasSystems]);
```

Comment:
We need to preserve this empty state.

Reply:
I added a ZeroState.js file that could be used in the element props for Route.
The only issue I have is the Pathways details throwing a 404 (I'm looking into why it happens), everything else works.